src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -791,7 +791,7 @@ private void mapEresource(List<CompletableFuture<?>> futures, Map<Mapping.Field,
 
     Optional.ofNullable(mappings.get(Mapping.Field.USER_LIMIT))
       .ifPresent(field -> futures.add(field.resolve(doc)
-        .thenAccept(o -> eresource.setUserLimit((Integer) o))
+        .thenAccept(o -> eresource.setUserLimit((String) o))
```

Comment:
we need also to increase `orders` interface to 13.0 inside ModuleDescriptorTemplate

Reply:
also please do it for other similar PRs like in mod-invoice. Also before merging we should contact UI devs to update ui modules as well. CC: @usavkov-epam 
Also we need to search other github repos like this one https://github.com/folio-org/mod-inventory/blob/master/descriptors/ModuleDescriptor-template.json and create necessary PRs. Without this next reference env build will fail